index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My first Portfolio</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+</head>
+<body>
+    <nav class="toolbar">
+        <h3>Moise Mulungu</h3>
+        <div class="bars">
+            <i class="fas fa-bars"></i>
+        </div>
+    </nav>
+    <header>
+        <h1 class="intro">Hey there.<br>
+            I'm Moise
+        </h1>
+        <h2>I'm a software developer</h2>
+        <p> I can help you build a product, feature or<br>
+            website Look through some of my work<br> 
+            experience! if you like what you see and<br>
+            have a project you needed code, do not<br>
+            hesitate to contact me.
+        </p>
+        <nav class="contacts">
+            <ul>
+                <li><a href="https://github.com/moise-mulungu/portfolio-mobile-version"><i class="fab fa-github"></i></a></li>
+                <li><a href="https://www.linkedin.com/in/mo%C3%AFse-mulungu-a939831b2/"><i class="fab fa-linkedin"></i></a></li>
+                <li><a href="https://twitter.com/moise_mulungu"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="https://www.facebook.com/moses.mulungu"><i class="fab fa-facebook"></i></a></li>
+            </ul>
+        </nav>
+
+    </header>
```

Comment:
- [ ] The container of the section is a `section` tag. Please change the `header` tag with a `section` tag.

- [ ] I think we are missing the icons here. As you can see in the image below:
The original design: 
![image](https://user-images.githubusercontent.com/60292761/148629616-9377d58d-8956-4f96-92be-2f2f33e6050d.png)
Your design:
![image](https://user-images.githubusercontent.com/60292761/148629632-a393aa4a-fbb3-4d6c-91c0-677eb82d5ed1.png)
There are a few websites for icons, you can check [fontawesome](https://fontawesome.com/v5.15/icons?d=gallery&p=2). You can also check [this link](https://www.w3schools.com/icons/) for more information about adding icons 😊 .

- [ ] Also, I think it's better if we add classes name for our HTML tags.